### PR TITLE
Reload config instead of doing hard reset

### PR DESCRIPTION
**Pull request by ton31337**
Reload config instead of doing hard reset.
@vinted/sre

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -55,7 +55,12 @@
               event: node[:exabgp][:hubot][:event]
             }
   mode 0755
-  notifies :restart, 'service[exabgp]'
+  notifies :run, 'execute[reload-exabgp-config]'
+end
+
+execute 'reload-exabgp-config' do
+  action :nothing
+  command 'kill -USR2 $(pgrep -f bgp.py) ; true'
```
Comment (benner) [inline recipes/default.rb]: pkill -USR2 -f bgp.py?

Comment (ton31337) [inline recipes/default.rb]: oh, really, how I forgot my favorite command (pkill) ;-)

**Comment by deees** [comment]
:+1:

**Comment by tomas-didziokas** [comment]
:+1:

**Pull request merged**